client/src/components/button/PageNationButton.jsx

```
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from './PageNationButton.module.css';
+import left from '../../assets/left.png';
+import right from '../../assets/right.png';
+
+export default function PageNationButton({ clickHandler, side = 'right' }) {
+  const icons = { left: left, right: right };
+
+  return (
+    <div className={styles.wrapper}>
+      <button className={styles.circle} onClick={clickHandler}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)